libraries/aws_ecs_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AWSECSService < AwsResourceBase
+  name 'aws_ecs_service'
+  desc 'Describes the specified services running in your cluster.'
+
+  example "
+    describe aws_ecs_service(cluster: 'test-cluster', services: 'test1') do
+      it { should exist }
+    end
+  "
+  def initialize(opts = {})
+    opts = { services: opts } if opts.is_a?(String)
+    super(opts)
+    validate_parameters(required: %i(services), require_any_of: %i(cluster))
```

Comment:
here cluster should be validated for `allow` and not `require_any_of`

Reply:
nice catch
Actually, Cluster us a required parameter will Update this

Reply:
no cluster is not a required parameter and that is why I have suggested to use `allow` it is an optional params as per the [docs](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/ECS/Client.html#describe_services-instance_method)